draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -366,187 +369,201 @@
 
     </section>
 
-    <section title=" Modules Tree Structure">
-
-      <t>This document defines a base Traffic Policy module, a QoS
-      specific Action module, separate policy modules for Scheduler,
-      Queuing and DiffServ policies and operational module for QoS
-      counters.</t>
-
-      <t>Traffic Policy module contains list of classifiers
-      identified by a classifier name. Each classifier MAY contain a
-      list of filters. When no filter is present in a classifier, it
-      matches all traffic.</t>
+    <section title="YANG Modules">
+      <t>
+      Modules defined in this draft import definitions from "Common
+      YANG Data Types" <xref target="RFC6991"/> and "A YANG Data Model
+      for Interface Management" <xref target="RFC8343"/>.
+      </t>
 
-      <t>This module also contains list of policy objects identified by a
-      policy name and policy type which MUST be provided. With different
-      values of policy types, each vendor MAY define their own construct of
-      policy for different QoS functionalities. Each vendor MAY augment
-      classifier entry in a policy definition with a set of actions.</t>
+      <section title="Traffic Policy Module">
 
-      <figure>
-	<name>ietf-traffic-policy tree diagram</name>
-        <artwork><![CDATA[
-INSERT_TEXT_FROM_FILE(../bin/ietf-traffic-policy@YYYY-MM-DD-sub-tree.txt)
+	<t>Traffic Policy module contains list of classifiers
+	identified by a classifier name. Each classifier MAY contain a
+	list of filters. When no filter is present in a classifier, it
+	matches all traffic.</t>
 
-    ]]></artwork>
-      </figure>
+	<t>This module also contains list of policy objects identified
+	by a policy name and policy type which MUST be
+	provided. Policy-type is of type identity and is populated in
+	a vendor specific manner. With different values of policy
+	types, each vendor MAY define their own construct of policy
+	for different QoS functionalities. Each vendor MAY augment
+	classifier entry in a policy definition with a set of
+	actions (Note, not clear what this sentence means).</t>
```

Comment:
This means the Traffic Policy Module is extendable.. you can avoid the sentence if you want..